modules/ROOT/pages/introduction/use-exchange.adoc

```diff
@@ -0,0 +1,186 @@
+= Use Exchange to Discover Connectors, Templates, and Examples
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Use Anypoint Exchange (Exchange) to discover connectors, templates, and examples, and to incorporate them into your app. Exchange serves as a repository for connectors, templates, and examples provided both by MuleSoft and by other organizations. Searches in Exchange apply only to the Exchange organization you are viewing.
```

Comment:
"Searches in Exchange apply only to the Exchange organization you are viewing." I think this is incorrect, because if you click "All assets" on the left in Exchange and run a search such as "amazon", it returns both search results from assets in the current Exchange organization and search results from assets in MuleSoft's Exchange (which are shown under the link "Provided by MuleSoft"). I recommend changing those two paragraphs to this:

Use Anypoint Exchange (Exchange) to discover connectors, templates, and examples, and to incorporate them into your app. Exchange serves as a repository for connectors, templates, and examples provided both by MuleSoft and by other organizations. Exchange enables both searching for assets in the Exchange organization you are viewing, and searching MuleSoft's public assets in the https://www.mulesoft.com/exchange/[Public Exchange].

Reply:
I had updated this to the following. I am sorry that this didn't show up when you looked at it:

Use Anypoint Exchange to discover integration assets such as connectors, templates, and examples and import them into your app or project. You can browse and search for assets built and curated by MuleSoft or assets available in your own organization's exchange.

To access assets in MuleSoft's Exchange, do either of the following:

* Go to https://www.mulesoft.com/exchange/.
* Go to your organization's Exchange and click the *Provided by MuleSoft* link in the left navigation panel.

To access assets in your own organization's Exchange, you must have the appropriate permission.

For more information about searching for assets in Exchange, see xref:exchange::to-find-info.adoc[Find Asset Information].

Reply:
I would change the second to last paragraph to be this: "To access your organization's Exchange, you must have access rights to that Exchange."

Reply:
Yes, this looks good! I slightly prefer the current version, "you must have the appropriate permission", but the other version sounds fine too.